packages/shared/lib/core/network/constants/default-chain-configurations.constant.ts

```diff
@@ -1,17 +1,18 @@
+import { DestinationNetwork } from '@core/layer-2'
 import { ChainType, NetworkId } from '../enums'
 import { ChainConfiguration } from '../types'
 
 export const DEFAULT_CHAIN_CONFIGURATIONS: Readonly<{ [id in NetworkId]?: ChainConfiguration }> = {
     [NetworkId.Shimmer]: {
         type: ChainType.Iscp,
-        name: 'ShimmerEVM',
+        name: DestinationNetwork.Shimmer,
```

Comment:
shouldnt it be `DestinationNetwork.ShimmerEvm`

Reply:
Good catch!